Example project/main.c

```diff
@@ -1,68 +1,137 @@
+/** 
+  ******************************************************************************
+  * @file    ws2812b_lld.h
+  * @brief   example project for WS2812B Library
+  * @author  https://github.com/j3qq4hch
+  * @author  https://github.com/mnemocron
+  * @date    2020-10-27
+  * 
+  * @note    IAR Embedded Workbench 3.11.1
+  *          shared components 8.3.2.5988
+  * @note    tested with RGBW LED type: IN-PI55QATPRPGPBPW-XX
+  * @note    STM8 must run at 16 MHz
+  * 
+  ******************************************************************************
+  */
+/* Includes ------------------------------------------------------------------*/
 #include <string.h>
 #include <stdio.h>
 #include "stm8s_conf.h"
 #include "ws2812B_fx.h"
```

Comment:
ws2812b_fx.h not ws2812B_fx.h - mind caps in file names. Doesn't compile on Unixes for this reason